packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -21,7 +21,17 @@ import type { BaseSqlExecuteArgs } from "../execute";
 import type { Argv } from "yargs";
 
 export function ApplyOptions(yargs: Argv): Argv<BaseSqlExecuteArgs> {
-	return Database(yargs);
+	return Database(yargs)
+		.option("local", {
+			describe:
+				"Execute commands/files against a local DB for use with wrangler dev",
+			type: "boolean",
+		})
+		.option("persist-to", {
+			describe: "Specify directory to use for local persistence (for --local)",
```

Comment:
Can we use something like https://yargs.js.org/docs/#implies to enforce the "for `--local`" relationship?

Reply:
Resolving pending followup